qiskit_sphinx_theme/furo/base/page.html

```diff
@@ -51,7 +53,8 @@
       </div>
       <label class="toc-overlay-icon toc-header-icon{% if furo_hide_toc %} no-toc{% endif %}" for="__toc">
         <div class="visually-hidden">Toggle table of contents sidebar</div>
-        <i class="icon"><svg><use href="#svg-toc"></use></svg></i>
+        {#- QISKIT Todo: change svg-toc with  Carbon icons -#}
```

Comment:
Ditto here on adding `{#- QISKIT CHANGE: start.` and `{#- QISKIT CHANGE: end. -#}`